temporian/implementation/numpy/data/event.py

```diff
@@ -51,16 +44,8 @@ def __init__(
         self.data = data
         self.sampling = sampling
 
-    def schema(self) -> Event:
-        return Event(
-            features=[
-                feature.schema() for feature in list(self.data.values())[0]
-            ],
-            sampling=self.sampling.names,
-        )
-
```

Comment:
What's the motivation behind removing the ability to create schemas from materialized data?

Reply:
From a design perspective, the semantic of a schema created by the data is not clear to me. Let brainstorm about it in our next meeting:

In the mean time, here are some thoughts:

- The graph defines the computation (which include schema i.e. input and output type of each step).
- The data is something that can be feed into the graph.
- The data feed by the user might be different from the data received by the graph. For example, the current code automatically read csv files when the data looks like a path. I suppose we will also implement automatic numerical conversions (e.g. the schema is a int8 but the user feed a int32; => The data is automatically converted to int8).